EnglishFilter/sections/specific.txt

```diff
@@ -2368,7 +2368,12 @@ heroero.com##.block-video > .table > .opt
 ||heroero.com/temp/2ava.jpg
 efukt.com##.efukt-widget-slider-nice-try-adblockers
 sportnews.to##span.mol-ads-label-container
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com,important
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com|onlinemschool.com,important
+!+ PLATFORM(android, ios)
```

Comment:
do you mean `AG for Android` or Content Blocker?

Reply:
For android